mixnode/src/commands/describe.rs

```diff
@@ -47,17 +47,25 @@ pub fn execute(matches: &ArgMatches) {
     let description = desc_buf.trim().to_string();
 
     let example_url = "https://mixnode.yourdomain.com".bright_cyan();
+    let example_location = "City: YourCity, Country: YourCountry";
```

Comment:
I'd change it to a more concrete example like "London, UK", as people will enter the literal string of "YourCity, YourCountry"

Reply:
Done.